Applications/HTTP/httpd-fs/main.js

```diff
@@ -83,18 +83,79 @@ templates['color2'] = `
         </table>
         <div style="width:200px; height:100px;margin:auto; background:rgb({{r_dec}},{{g_dec}},{{b_dec}});"></div>
 `;
+// Temolate for thermo3 shows two lists one with the actial values red from the eval Board Memory one one from stored values
+templates['thermo3'] = `
+        <h2>Thermo-3 Clickboard</h2>
+        <p>
+        	<h3>dLAN Green PHY Werte</h3>
+        	<table class="table table-striped">
+        		<tr>
+        			<td>Aktuelle Temperatur</td>
+        			<td>{{temp}}&degC</td>
+        		</tr>
+        		<tr>
+        			<td>H&oumlchste Temperatur</td>
```

Comment:
I'm not sure wether it's necessary to escape HTML entities, but it must be `&ouml;`